proto/p4/v1/p4runtime.proto

```diff
@@ -429,10 +429,10 @@ message MulticastGroupEntry {
 // packet_length_bytes field is 0, no truncation on the clone(s) will be
 // performed.
 message CloneSessionEntry {
-  uint64 session_id = 1;
+  uint32 session_id = 1;
   repeated Replica replicas = 2;
-  uint64 class_of_service = 3;
-  uint64 packet_length_bytes = 4;
+  uint32 class_of_service = 3;
+  int32 packet_length_bytes = 4;
```

Comment:
That's a good question. I seem to remember that @wmohsin recommended that we use `int32` (as opposed to `uint32`) for integer values even if the P4Runtime spec states that the entities cannot be negative. Which is why p4runtime.proto consistently uses `int32` for length / size fields. On the other hand we use `uint32` for fields which encode a specific bit pattern (ids).
I cannot find anything in the Protobuf documentation stating that `int32` should be used, so maybe I'm making that whole thing up :) Or maybe using `int32` helps avoid errors on the client side (e.g. if the client tries to pass a negative number, it would get an error from the server instead of the server using a large integer value).
I'm going to merge this for now as it is consistent with the rest of p4runtime.proto, but we should definitely bring this up at the next WG meeting and if there is a reason for using `int32` for unsigned entities, maybe we should add a note somewhere.